backend/src/api.ts

```diff
@@ -207,6 +214,23 @@ loginCheckedPost('/deleteTeamEvent', async (req, user) => ({
   team: await deleteTeamEvent(req.body, user)
 }));
 
+// Team Events Proof Images
+loginCheckedGet('/getEventProofImageSignedURL', async (req, user) => ({
+  url: await setProofImage(user, req.body.name)
+}));
+loginCheckedGet('/getEventProofImage', async (req, user) => ({
+  url: await getProofImage(user, req.body.name)
+}));
+loginCheckedGet('/getAllEventProofsForMember', async (_, user) => ({
+  url: await allProofImagesForMember(user)
+}));
+loginCheckedPost('/deleteEventProofImage', async (req, user) => ({
+  image: await deleteProofImage(user, req.body.name)
+}));
+loginCheckedPost('/deleteAllEventProofsForMember', async (_, user) => ({
+  image: await deleteProofImagesForMember(user)
+}));
```

Comment:
I don't think we need these as API endpoints. Images should only get deleted when a request is either approved or rejected which is handled by the team event API.

Reply:
Done!